src/VisualStudio/Core/Def/InheritanceMargin/MarginGlyph/TargetMenuItemViewModel.cs

```diff
@@ -23,19 +24,33 @@ internal class TargetMenuItemViewModel : MenuItemViewModel
         /// </summary>
         public DefinitionItem.DetachedDefinitionItem DefinitionItem { get; }
 
+        public ImageMoniker LanguageMoniker { get; }
+
         // Internal for testing purpose
         internal TargetMenuItemViewModel(
             string displayContent,
             ImageMoniker imageMoniker,
-            DefinitionItem.DetachedDefinitionItem definitionItem) : base(displayContent, imageMoniker)
+            DefinitionItem.DetachedDefinitionItem definitionItem,
+            ImageMoniker languageMoniker) : base(displayContent, imageMoniker)
         {
             DefinitionItem = definitionItem;
+            LanguageMoniker = languageMoniker;
         }
 
         public static TargetMenuItemViewModel Create(InheritanceTargetItem target, string displayContent)
-            => new(
+        {
+            var languageGlyph = target.LanguageName switch
+            {
+                LanguageNames.CSharp => Glyph.CSharpFile,
+                LanguageNames.VisualBasic => Glyph.BasicFile,
+                _ => throw ExceptionUtilities.UnexpectedValue(target.LanguageName),
```

Comment:
don't love this.  can we leave this out?  consider, for example, if F# or JS/TS use this.

Reply:
Then I will move this to the OOP side (which is own by us)
let InheritanceTargetItem contains a glyph property, if the other language wants to use this and they can give us the glyph

Reply:
Yup.  That would be perfect.